lib/github/api.nit

```diff
@@ -1602,13 +956,301 @@ end
+		var user = deserialize_attribute("user").as(User)
+		var repo = deserialize_attribute("repo").as(Repo)
+		return new PullRef(labl, ref, sha, user, repo)
+	end
+
+	# Deserialize an issue event
+	fun deserialize_issueevent: IssueEvent do
+		var html_url = deserialize_attribute("html_url").as(nullable String)
+		var id = deserialize_attribute("id").as(Int)
+		var actor = deserialize_attribute("actor").as(User)
+		var created_at = deserialize_attribute("created_at").as(String)
+		var iso_created_at = new ISODate.from_string(created_at)
+		var event = deserialize_attribute("event").as(String)
+		var commit_id = deserialize_attribute("commit_id").as(nullable String)
+		var labl = deserialize_attribute("label").as(nullable Label)
+		var assignee = deserialize_attribute("assignee").as(nullable User)
+		var milestone = deserialize_attribute("milestone").as(nullable Milestone)
+		var rename = deserialize_attribute("rename").as(nullable RenameAction)
+		return new IssueEvent(html_url, id, actor, iso_created_at, event, commit_id, labl, assignee, milestone, rename)
+	end
+
+	# Deserialize an issue comment
+	fun deserialize_issue_comment: IssueComment do
+		var html_url = deserialize_attribute("html_url").as(nullable String)
+		var id = deserialize_attribute("id").as(Int)
+		var user = deserialize_attribute("user").as(User)
+		var created_at = deserialize_attribute("created_at").as(String)
+		var iso_created_at = new ISODate.from_string(created_at)
+		var updated_at = deserialize_attribute("updated_at").as(nullable String)
+		var iso_updated_at = null
+		if updated_at != null then iso_updated_at = new ISODate.from_string(updated_at)
+		var body = deserialize_attribute("body").as(String)
+		var issue_url = deserialize_attribute("issue_url").as(String)
+		return new IssueComment(html_url, id, user, iso_created_at, iso_updated_at, body, issue_url)
+	end
+
+	# Deserialize a commit comment
+	fun deserialize_commit_comment: CommitComment do
+		var html_url = deserialize_attribute("html_url").as(nullable String)
+		var id = deserialize_attribute("id").as(Int)
+		var user = deserialize_attribute("user").as(User)
+		var created_at = deserialize_attribute("created_at").as(String)
+		var iso_created_at = new ISODate.from_string(created_at)
+		var updated_at = deserialize_attribute("updated_at").as(nullable String)
+		var iso_updated_at = null
+		if updated_at != null then iso_updated_at = new ISODate.from_string(updated_at)
+		var body = deserialize_attribute("body").as(String)
+		var commit_id = deserialize_attribute("commit_id").as(String)
+		var position = deserialize_attribute("position").as(nullable Int)
+		var line = deserialize_attribute("line").as(nullable Int)
+		var path = deserialize_attribute("path").as(nullable String)
+		return new CommitComment(html_url, id, user, iso_created_at, iso_updated_at, body, commit_id, position, line, path)
+	end
+
+	# Deserialize a review comment
+	fun deserialize_review_comment: ReviewComment do
```

Comment:
is there not way to simplify (or generate) the code of the `deserialize_*` methods?